guides/theming.md

```diff
@@ -235,6 +238,95 @@ your project's `angular.json` file][adding-styles].
 
 [adding-styles]: https://angular.io/guide/workspace-config#styles-and-scripts-configuration
 
+#### Theming dimensions
+
+Angular Material themes are divided along several dimensions:
+
+Base
+: Common base styles for the design system. These styles don't change based on your configured
+colors, typography, or density, so they only need to be included once per application. These
+styles may include things such as border-radius, border-width, etc. All components have a base
+mixin that can be used to include its base styles. (For example,
```

Comment:
Should we say something about only most apps needing to import them once, but that you can still include it multiple times if you want to change the base styles for different parts of your app?

Reply:
Well the thing is right now you can't really change it. The base styles are basically values for all of the tokens that don't depend on color, typography, or density, but may vary between M2, M3, or some future Material Design version. Since currently M2 is the only one available, the mixin always emits the same values.

In the future there could maybe be a use case for it, if some app wanted to display M2 vs M3 conditionally

Reply:
Another future use case is maybe once we have a Sass API to allow per-token customizations, but again thats not something we have now, so for now it just feels simplest to say "include it once"